src/bin/rdcore/rootmap.rs

```diff
@@ -30,6 +30,20 @@ use crate::cmdline::*;
 /// composefs, the mount of / and /sysroot will be distinct.
 const PHYSICAL_ROOT_MOUNT: &str = "sysroot";
 
+fn is_on_multipath(device: &str) -> Result<bool> {
+    let on_mpath = find_parent_devices(device)?
```

Comment:
Hmm, I don't think the logic is quite right here. We shouldn't check if any of the parent devices is multipath, but rather whether the immediate parent is.

We already have `backing_devices` which is a list of all the parent devices in order from lowest to highest (ending with the final device containing the rootfs itself). So... I think basically:
- get the _before last_ device in that Vec
- call `lsblk_single`, check if `TYPE` is `mpath`

?